scripts/pylintrc_default

```diff
@@ -200,7 +200,7 @@ no-space-check =
 # Maximum number of lines in a module
 # DEFAULT:  max-module-lines=1000
 # RATIONALE:  API-mapping
-max-module-lines=1593
+max-module-lines=1200
```

Comment:
It was `1500` but I [broke things][1] in #1557 with the promise that I'd restore order. I reset to 1200 since we didn't have anything exceeding it (it was at 1500 for test modules but now `run_pylint.py` overrides for test modules).

https://github.com/GoogleCloudPlatform/gcloud-python/pull/1557/files#diff-e0c0ad1427ffa06c687aab23895bad5a